.github/workflows/e2e.yml

```diff
@@ -3,6 +3,8 @@ name: e2e
 on:
   push:
     branches: [main]
+  pull_request:
+    branches: [ main ]
```

Comment:
We usually make PR to main. In short, e2e test will run each branch.
But it's too much, I think.

How about run e2e before publish new version cli ?

My proposal is 
1. e2e core logic moves to common file use by composite feature
  - https://github.blog/changelog/2021-08-25-github-actions-reduce-duplication-with-action-composition/
2. e2e test and python-publish use it.

Reply:
removed the change since making a composition looks out of scope for the purpose of the PR.
https://github.com/launchableinc/cli/pull/393/commits/d5ec7704fce3b154eb11bc3bb3a46c09a4cffc92